### PR TITLE
D8 nid 961

### PR DESCRIPTION
**Pull request by omahm**

### CODE REVIEW

File: migrate_nidirect_file/config/install/migrate_plus.migration.upgrade_d7_file.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-uuid: 36b1f2a8-ae69-4cdb-917e-34680a84d9f8
+quuid: 36b1f2a8-ae69-4cdb-917e-34680a84d9f8
```
Comment (johangant) [inline migrate_nidirect_file/config/install/migrate_plus.migration.upgrade_d7_file.yml:1]: Not sure if a typo? The following value looks the same so I'd say it's likely :)

File: migrate_nidirect_node/migrate_nidirect_node_nidirect_contact/src/Plugin/migrate/source/NIDirectContactNodeRevisionSource.php [hunk 2]
```diff
@@ -18,6 +18,7 @@
  *
  * @MigrateSource(
  *   id = "nidirect_contact_node_revision_source",
+ *   source_module = "node"
```
Comment (johangant) [inline migrate_nidirect_node/migrate_nidirect_node_nidirect_contact/src/Plugin/migrate/source/NIDirectContactNodeRevisionSource.php:21]: I remember migrate was sometimes grumpy about this being missing in the past. I guess it's now settled on needing to be there

**Review by johangant — APPROVED** [review]
Great stuff, nice to see all the past commands now centralised within Drush. The majority of the other changes look like appropriate and repeating YAML annotations so weren't too taxing to get through

File: migrate_nidirect_file/config/install/migrate_plus.migration.upgrade_d7_file.yml (hunk 1 above)
Comment (omahm) [reply migrate_nidirect_file/config/install/migrate_plus.migration.upgrade_d7_file.yml]: It is indeed, good catch

File: migrate_nidirect_node/migrate_nidirect_node_nidirect_contact/src/Plugin/migrate/source/NIDirectContactNodeRevisionSource.php (hunk 2 above)
Comment (omahm) [reply migrate_nidirect_node/migrate_nidirect_node_nidirect_contact/src/Plugin/migrate/source/NIDirectContactNodeRevisionSource.php:21]: yeah, it only really started complaining about it once I'd reinstalled the modules and ran migrate status

**Pull request merged**